### PR TITLE
Initial commit for Cluster API app

### PR DESCRIPTION
**Pull request by tuommaki**
This PR:

- adds/changes/removes etc

### Testing

Description on how {APP-NAME} can be tested.

- [ ] fresh install works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM
- [ ] upgrade from previous version works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM

#### Other testing

Description of features to additionally test for {APP-NAME} installations.

- [ ] check reconciliation of existing resources after upgrading
- [ ] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,10 +8,10 @@ workflows:
       - architect/push-to-app-catalog:
           context: "architect"
           # executor: "app-build-suite" # uncomment this if you want automatic metadata generation
-          name: "package and push {APP-NAME} chart"
+          name: "package and push cluster-api chart"
           app_catalog: "giantswarm-playground-catalog"
           app_catalog_test: "giantswarm-playground-test-catalog"
-          chart: "{APP-NAME}"
+          chart: "cluster-api"
```
Comment (tuommaki) [inline .circleci/config.yml:14]: Is this the right name of the chart?

File: README.md [hunk 2]
```diff
@@ -1,64 +1,5 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/{APP-NAME}-app.svg?style=shield)](https://circleci.com/gh/giantswarm/{APP-NAME}-app)
+[![CircleCI](https://circleci.com/gh/giantswarm/cluster-api-app.svg?style=shield)](https://circleci.com/gh/giantswarm/cluster-api-app)
 
-# {APP-NAME} chart
+# cluster-api chart
 
-Giant Swarm offers a {APP-NAME} App which can be installed in workload clusters.
```
Comment (tuommaki) [inline README.md:5]: This ain't workload cluster app but the control plane for Cluster API management cluster components.

File: helm/cluster-api/templates/_helpers.tpl [hunk 3]
```diff
@@ -1,33 +1,24 @@
...
+
 {{/*
 Common labels
 */}}
 {{- define "labels.common" -}}
 {{ include "labels.selector" . }}
-app.kubernetes.io/name: {{ .Values.name }}
-app.kubernetes.io/instance: {{ .Release.Name | quote }}
-app.giantswarm.io/branch: {{ .Values.project.branch | replace "#" "-" | replace "/" "-" | replace "." "-" | trunc 63 | trimSuffix "-" | quote }}
+app.giantswarm.io/branch: {{ .Values.project.branch | quote }}
```
Comment (tuommaki) [inline helm/cluster-api/templates/_helpers.tpl:17]: These templates are copied & adapted from earlier `cluster-api-<foobar>-app` repos.

File: .circleci/config.yml (hunk 1 above)
Comment (fiunchinho) [reply .circleci/config.yml:14]: I don't see why not.

Comment (ubergesundheit) [reply .circleci/config.yml:14]: yeah looks good!

**Review by fiunchinho — APPROVED** [review]
Let's get the ball moving!

File: helm/cluster-api/templates/_helpers.tpl (hunk 3 above)
Comment (ubergesundheit) [reply helm/cluster-api/templates/_helpers.tpl:17]: that wont work, values.yaml does not have `project.branch`

**Review by ubergesundheit — APPROVED** [review]
looks good from a first glance, make sure you set up circle

**Pull request merged**